### ISSUE TITLE
[M0-Dev] Clase predicción

### ISSUE THREAD

**Issue by manujurado1** [ignaciotitos/IV---HealthForAll#26]
Con tal de cumplir la HU #6 y saber el valor del dinero en el futuro, he visto que la lógica de negocio se enfoca en hacer una predicción, por lo que necesito saber que datos se necesitan para poder aplicar la lógica de negocio y hacer una predicción.
Según tengo entendido, la lógica de negocio se llevaría a cabo haciendo una regresión usando tasas de inflación de años anteriores para predecir la inflacion en un futuro, ¿estoy en lo cierto?.
Si esto es así, considero que lo necesario para poder hacer una predicción sería una lista de tasas de inflación. ¿Piensas igual?

**Comment by ignaciotitos** [comment]
Sí. Para hacer la regresión va a ser necesaria las tasas de inflación.

**Comment by manujurado1** [comment]
Me ha surgido una nueva duda respecto a como aplicar la lógica de negocio en referencia a la predicción. Para realizar la regresión y efectuar la predicción, necesitaremos saber para que fecha futura querrá el usuario ver dicha predicción verdad?
¿O la idea es mostrar predicciones de diferentes momentos futuros?

**Comment by manujurado1** [comment]
Voy a dar por hecho que esta información es necesaria para la predicción, si por lo que sea no te encajara con tu idea de aplicar la lógica de negocio pues ya lo modifico
